### PR TITLE
Makes religious items better

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
add: Chaplains who get their hands on clockwork materials have reported disappointment that they dissolve into slag.
/:cl:

Other then that just some back-end work for reclaiming the station for SSreligion.Bible_deity_name.

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -61,7 +61,8 @@
 	var/datum/devilinfo/devilinfo //Information about the devil, if any.
 	var/damnation_type = 0
 	var/datum/mind/soulOwner //who owns the soul.  Under normal circumstances, this will point to src
-
+	var/isholy == 0 //is this person a chaplen or specail role allowed to use bibles
```
Comment (RemieRichards) [inline code/datums/mind.dm]: ```=``` not ```==```, ```FALSE``` not ```0``` (same for ```TRUE``` and ```1```)

File: code/game/gamemodes/clock_cult/clock_items/clock_components.dm [hunk 2]
```diff
@@ -11,8 +11,15 @@
 
 /obj/item/clockwork/component/pickup(mob/living/user)
 	..()
-	if(iscultist(user) || (user.mind && user.mind.assigned_role == "Chaplain"))
+	if(iscultist(user) || (user.mind && user.mind.isholy == 1))
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_items/clock_components.dm]: ```(user.mind && yuser.mind.isholy)```

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 3]
```diff
@@ -322,12 +322,10 @@
 
 /obj/item/weapon/nullrod/carp/attack_self(mob/living/user)
 	if(used_blessing)
-		return
-	if(user.mind && (user.mind.assigned_role != "Chaplain"))
-		return
-	user << "You are blessed by Carp-Sie. Wild space carp will no longer attack you."
-	user.faction |= "carp"
-	used_blessing = TRUE
+	else if(user.mind && (user.mind.isholy == 1))
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/holy_weapons.dm]: no point in this being an else if, it's never ran if the above is true anyway.

File: code/datums/mind.dm (hunk 1 above)
Comment (Supermichael777) [inline code/datums/mind.dm]: your right

**Pull request closed without merging**